### DISCUSSION TITLE
Is it safe for a vault in OneDrive sync folder to enable remotely save?

### DISCUSSION THREAD

**Discussion by scottctd** [remotely-save/remotely-save#510] [Q&A] [answered]
Basically the title.

I have my vault in my OneDrive Personal folder.

For the sake of sync across mobile devices, I also enabled remotely-save on the vault.

Is this safe? Will I lose any files/get any chaos?

**Comment by fyears** [comment, marked as answer]
like this?

```
mobile
 |
 | remotely save
 |
onedrive (website)
 |
 | onedrive official software
 |
pc
```

I am not sure because i don't know how onedrive official software works. And of course I don't recommend doing so.

Basically remotely save works by comparing last modified time, so you should do some test to check the how the modified time changes.

And surely, **always backup your vault**!!!

**Reply by scottctd** [reply]
Sure and I may update if there are any discrepancies caused by my setup.

What I really want to achieve is 
Multiple PCs sync the vault using onedrive itself
Changes from PC are sync to mobile through remotely-save because IOS obsidian doesn't support onedrive.

I know this sounds stupid, but it may work.

Thank you for making this great plugin.
